.github/workflows/README.md

```diff
@@ -0,0 +1,25 @@
+# Workflow Directory
```

Comment:
This is really neat. Should it be linked to from the root `README.md` to make it more discoverable?